foqus_lib/gui/main/settingsFrame.py

```diff
@@ -71,6 +71,18 @@ def __init__(self, dat, parent=None):
         self.TestTurb.clicked.connect(self.turbineTest)
         self.testLite.clicked.connect(self.turbineLiteTest)
         self.changePort.clicked.connect(self.updateTurbineLitePort)
+        self.runMethodCombo.currentIndexChanged.connect(
+            self.displayAvailablityWarning)
+
+    def displayAvailablityWarning(self):
+        """
+        Warn that changing the Turbine server means that a different set of
+        simulations may be available and they may not match your flowsheet
+        """
+        QMessageBox.warning(self, "Warning", "You are changing the Turbine server"
+            " connection.  The the new server may not have the simulations or"
```

Comment:
typo "The the new server"
```suggestion
            " connection.  The new server may not have the simulations or"
```